examples/sign_and_verify_example.js

```diff
@@ -0,0 +1,18 @@
+const { ec } = require("elliptic");
+const secp256k1 = new ec("secp256k1");
+const { signDeploy, verifyDeploy } = require("@tgrospic/rnode-grpc-js");
+require("..");
+const key = secp256k1.genKeyPair();
+// const key = '1bf36a3d89c27ddef7955684b97667c75454317d8964528e57b2308947b250b0'
+const sampleRholangCode = "Nil";
+const deployData = {
+  term: sampleRholangCode,
+  phloprice: 1,
+  phlolimit: 10e3,
+  validafterblocknumber: 0,
```

Comment:
Please add `timestamp` as `Date.now()`. I'll remove default in the future versions of _rnode-grpc-js_, because it's global dependency on Date object.